contrib/golang/filters/http/test/test_data/routeconfig/filter.go

```diff
@@ -5,22 +5,12 @@ import (
 )
 
 type filter struct {
+	api.DefaultStreamFilter
```

Comment:
maybe PassThroughStreamFilter could be better? it could be more meaningfull for users.
there is also `Http::PassThroughFilter` in C++.